src/module/dy/util.js

```diff
@@ -24,7 +24,11 @@ module.exports = ({
   };
   const sets = Object.entries(attributes).filter(([_, v]) => v.type === 'set').map(([k]) => k);
   const mergeAttributes = MergeAttributes(sets);
-  const generateUpdateItem = GenerateUpdateItem(sets);
+  const generateItemMap = ['update', 'put', 'delete'].reduce((prev, cur) => {
+    // eslint-disable-next-line no-param-reassign
+    prev[cur] = GenerateItem(cur, sets);
+    return prev;
+  }, {});
```

Comment:
```suggestion
  const generateItemMap = {
    'update': GenerateItem('update', sets),
    ...
  };
```
don't make this so complicated :)